src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,115 @@
+
+.cards {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  @media screen and (width:1200px) {
+    .cards {
```

Comment:
Remove the nested '.cards' selector as it's not necessary, and update the margin properties directly within the media query.